File: exercicios/2486/2486.py
vit_c_min = 110
vit_c_max = 130
total = 0
while True:
    t = int(input())

    if t == 0:
        break

    for i in range(t):
        
        v = input().split()

        qt = int(v[0])
        op = ' '.join(v[1:])

        if op == 'suco de laranja':
            total += 120 * qt

        elif op == 'morango freco':
            total += 85 * qt
        
        elif op == 'mamao':
            total += 85 * qt

        elif op == 'goiaba vermelha':
            total += 70 * qt

        elif op == 'manga':
            total += 56 * qt
        
        elif op == 'laranja':
            total += 50 * qt

        elif op == 'brocolis':
            total += 34 * qt

    if total >= vit_c_min and total <= vit_c_max:
        print(total,'mg')
        total = 0
        continue

    elif total > vit_c_max:
        diminuir = total - vit_c_max
        print('Menos {} mg'.format(diminuir))
        total = 0
        continue

    elif total < vit_c_min:
        aumentar = vit_c_min - total
        print('Mais {} mg'.format(aumentar))
        total = 0
        continue

File: exercicios/JavaScript/2486/2486.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

let n = lines.shift();
let soma = 0;

while(n != 0){
    for(let i=0;i<n;i++){
        let [qt,fruta] = lines.shift().trim().split(' ');

        switch(fruta){
            case "suco":
                soma += parseInt(qt) * 120;
                break;
            
            case "morango":
                soma += parseInt(qt) * 85;
                break;

            case "mamao":
                soma += parseInt(qt) * 85
                break;

            case "goiaba":
                soma += parseInt(qt) * 70;
                break;

            case "laranja":
                soma += parseInt(qt) * 50;
                break;

            case "manga":
                soma += parseInt(qt) * 56;
                break;

            case "brocolis":
                soma += parseInt(qt) * 34;
                break;
        }
    }
    if(soma >= 110 && soma <= 130){
        console.log(`${soma} mg`);
    }else if( soma > 130){
        console.log(`Menos ${(130 - soma) * -1} mg`);
    }else if(soma < 110){
        console.log(`Mais ${110 - soma} mg`);
    }

    soma = 0;
    n = lines.shift();
}